nucliadb_utils/nucliadb_utils/audit/basic.py

```diff
@@ -41,12 +41,23 @@ async def search(
         self,
         kbid: str,
         user: str,
+        client: int,
         origin: str,
         search: SearchRequest,
         timeit: float,
         resources: int,
     ):
         logger.debug(f"SEARCH {kbid} {user} {origin} ''{search}'' {timeit} {resources}")
 
+    async def suggest(
+        self,
+        kbid: str,
+        user: str,
+        client: int,
+        origin: str,
+        timeit: float,
```

Comment:
what about `duration` as a better name?

Reply:
Agree, let me check which consequences may have this change on the receiving side of this events

Reply:
We have some places that make use of this field, so i'll leave it like this for now.